apps/molgenis-components/vite.config.js

```diff
@@ -1,53 +1,66 @@
-import { defineConfig } from "vite";
-import { createVuePlugin } from "vite-plugin-vue2";
-import path from "path";
-import docTagPlugin from "./docs-plugin.js";
+import {defineConfig} from 'vite';
```

Comment:
I propose we NOW change the defaults back, will do pull request in 5 mins so this silly changes don't happen